packages/components/src/__future__/Tag/_docs/Tag.stickersheet.stories.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "react"
+import { Meta, StoryFn } from "@storybook/react"
+import { TagIcon } from "~components/Icons"
+import { StickerSheet } from "~storybook/components/StickerSheet"
+import { Tag } from "../Tag"
+
+type ListProps = {
+  children: React.ReactNode
+}
+const List = ({ children }: ListProps): JSX.Element => (
+  <div className="flex flex-col gap-12 items-start">{children}</div>
+)
+
+export default {
+  title: "KAIO-staging/Tag",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+  },
+} satisfies Meta
+
+const StickerSheetTemplate: StoryFn = () => (
+  <StickerSheet heading="Tag">
+    <StickerSheet.Header headings={["Label Only", "Icon"]} />
+    <StickerSheet.Body>
+      <StickerSheet.Row>
```

Comment:
Nitpick: You could iterate over an array of colours, creating a row for each colour.

```
<StickerSheet.row rowTitle={colour}>
  <Tag color={colour}> ...
  <Tag color={colour} Icon={<TagIcon role="presentation" />}> ...
</StickerSheet.row>
```

Reply:
Agreed with Geoff. Then you can get rid of that `List` component at the top of the file.